scripts/extractMap.js

```diff
@@ -0,0 +1,28 @@
+const fs = require("fs");
+
+const LAYER_NAME = "Collides";
+
+const layerCollides = require("../public/assets/map/map.json").layers.find(
+  (layer) => layer.name === LAYER_NAME
+);
+
+const arr = [];
+
+for (let j = 0, m = 0; j < layerCollides.width; j += 1) {
+  arr.push([]);
+  for (let k = 0; k < layerCollides.height; k += 1) {
+    const tileId = layerCollides.data[m];
+    arr[j][k] = tileId === 0 ? -1 : tileId;
+    m += 1;
+  }
+}
+
+fs.writeFile(
+  "./public/assets/map/map.js",
```

Comment:
Maybe it should be exported to a separate file?

Reply:
Yup, maybe. ATM all files from ./public are copied to ./dist. We could clean it up a bit.

Reply:
A separate issue for this would be nice

Reply:
Here you go - #59